src/Sylius/Bundle/ResourceBundle/spec/Controller/ParametersParserSpec.php

```diff
@@ -25,165 +24,119 @@
  */
 final class ParametersParserSpec extends ObjectBehavior
 {
-    function let(ContainerInterface $container, ExpressionLanguage $expression)
+    function it_implements_parameters_parser_interface()
     {
-        $this->beConstructedWith($container, $expression);
+        $this->beConstructedWith(new Container(), new ExpressionLanguage());
+        $this->shouldImplement(ParametersParserInterface::class);
     }
 
-    function it_implements_parameters_parser_interface()
+    function it_should_parse_string_parameters()
```

Comment:
We don't use `it_should_...` convention for a very long time, why use it here in spite of simple `it_parses_...`?